graphql-kickstart-spring-boot-autoconfigure-graphql-annotations/src/main/java/graphql/kickstart/graphql/annotations/GraphQLAnnotationsAutoConfiguration.java

```diff
@@ -179,30 +195,32 @@ private void setQueryResolverClass(
    * This is required, because normally implementations of interfaces are not explicitly returned by
    * any resolver method, and therefor not added to the schema automatically.
    *
-   * All interfaces are considered GraphQL interfaces if they are declared in the configured package
-   * and have at least one {@link GraphQLField}-annotated methods.
+   * <p>All interfaces are considered GraphQL interfaces if they are declared in the configured
```

Comment:
Is the closing tag missing intentionally?

Reply:
The google-java-format plugin made this change automatically actually when I ran `./gradlew googleJavaFormat`. So it seems to have been coded this way intentionally by Google.